integration-manifest.json

```diff
@@ -285,20 +306,24 @@
                 "HasPrivateKey": false,
                 "OnAdd": false,
                 "OnRemove": false,
-                "OnReenrollment": true
-              }
+                "OnReenrollment": false
+              },
+              "DependsOn": "",
+              "DefaultValue": "",
+              "Options": ""
             }
           ],
           "PasswordOptions": {
             "EntrySupported": false,
             "StoreRequired": false,
             "Style": "Default"
           },
-          "StorePathValue": "[\"My\",\"WebHosting\"]",
+          "StorePathType": "",
+          "StorePathValue": "",
```

Comment:
Not sure if you're still working on this @rcpokorny, but for IISU we're going to want to restrict the store paths to My and WebHosting via a multiple-choice type.